components/ui/Table/OrderableTable.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { DndContext } from '@dnd-kit/core';
+import { TableBase } from './TableBase';
+import { createScaleModifier } from '../../../utils/scale-modifier';
+import { useContentDimensions } from '../../../hooks/use-content-dimensions';
+
+export function OrderableTable(props) {
+  const [isDragging, setIsDragging] = useState(false);
+  const { onReorder } = props;
+
+  const contentDimensions = useContentDimensions();
+  const scaleDragger = createScaleModifier(contentDimensions);
+
+  const handleDragStart = () => {
+    setIsDragging(true);
+  };
+
+  const handleDragEnd = ({ active, over }) => {
+    setIsDragging(false);
+    onReorder({ from: active.id, to: over?.id });
+  };
+
+  return (
+    <DndContext
+      onDragStart={handleDragStart}
+      onDragEnd={handleDragEnd}
+      modifiers={[scaleDragger]}
+    >
+      <TableBase isDragging={isDragging} {...props} />
```

Comment:
Do any passed children here need to be explicitly passed as a {children} block inside the TableBase, in order to be picked up, or can they just be another prop that gets handed down like every other? I'm always a bit unsure about what is an implicit prop and what is an explicit one?